deployment_script.py

```diff
@@ -3,11 +3,11 @@
 import os
 from text_based_scenarios.convert_yaml_to_json_config import main as generate_textbased_configs
 from text_based_scenarios.add_patient_pictures import main as add_pictures
```

Comment:
Don't think we need lines 4 and 5 here any more.

Reply:
removed, thanks!